src/sql/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -356,6 +356,11 @@ configurationRegistry.registerConfiguration({
 			'minimum': 10,
 			'description': localize('notebook.maxRichTextUndoHistory', "The maximum number of changes stored in the undo history for the notebook Rich Text editor.")
 		},
+		'notebook.renderTablesInHtml': {
+			'type': 'boolean',
+			'default': false,
+			'description': localize('notebook.renderTablesInHtml', "Display tables in HTML format in text cells instead of converting to markdown tables.")
```

Comment:
I feel like we need to be more explicit here about what's actually happening - because it's less that it's "displaying" the table in HTML format but rather converting the actual table text itself into either markdown or HTML. 

@chlafreniere Thoughts on this?

Reply:
How about "Display and save" instead of just "Display"?